src/components/DayPickerRangeController.jsx

```diff
@@ -819,6 +820,8 @@ export default class DayPickerRangeController extends React.Component {
     } = this.props;
 
     const { phrases, visibleDays } = this.state;
+    const initialVisibleMonthThunk =
+      initialVisibleMonth || (startDate ? () => startDate : () => moment());
```

Comment:
This looks great! This is definitely a bug (I originally didn't catch is because the wrapper I'm using has an `initialVisibleMonth` default prop).